src/app/settings/settings-routing.module.ts

```diff
@@ -3,9 +3,10 @@ import { RouterModule, Routes } from '@angular/router'
 import { SettingsComponent } from './components/settings/settings.component'
 import { ClearCacheComponent } from './components/clear-cache/clear-cache.component'
 
-export enum SettingsPaths {
+export enum SettingsPath {
   SETTINGS = '',
-  KEYBOARD_MAPPINGS = 'keyboard-mappings',
+  KEYBOARD_MAPPINGS = 'keyboard-mapping',
```

Comment:
`KEYBOARD_MAPPINGS` is plural while `keyboard-mapping` is singular.

Reply:
Nice catch. It's been updated